pgml-extension/pgml_extension/model.py

```diff
@@ -753,6 +753,18 @@ def predict(self, data: list):
         return y[0]
 
 
+def snapshot(
```

Comment:
Why define this at the root level, when it's just a pass through to the class?

Reply:
Excellent question...

Reply:
Will clean up later. I'm still thinking about this one.